modules/layers/src/column-layer/column-layer.js

```diff
@@ -115,28 +125,50 @@ export default class ColumnLayer extends Layer {
     });
   }
 
-  _getModel(gl) {
-    return new Model(
-      gl,
-      Object.assign({}, this.getShaders(), {
-        id: this.props.id,
-        geometry: this.getGeometry(this.props.diskResolution),
-        isInstanced: true,
-        shaderCache: this.context.shaderCache
-      })
-    );
+  _getModels(gl) {
+    const {id, filled, extruded, diskResolution} = this.props;
+
+    let filledModel;
+    let strokeModel;
+
+    if (filled) {
+      filledModel = new Model(
```

Comment:
What are the difference between these two models? Can we reuse the same one and draw with different draw modes?

Reply:
I thought of defining both the models by their traits. It seemed more readable. But yes, I could reuse the same one model with different `draw modes` with the conditions `wireframe` and `filled`.

Reply:
@kaushikb11 

You can with different draw modes, if your geometry is correctly filled and stroked.